### PR TITLE
RC - D7 updates (LEB-1649, LEB-1341, LEB-1452)

### PR DESCRIPTION
**Pull request by ynx**

### CODE REVIEW

File: acquia_lift.context.inc [hunk 1]
```diff
@@ -80,8 +80,6 @@ function _acquia_lift_get_page_context($node = NULL, $primary = FALSE) {
     $page_context['published_date'] = $node->created;
     $page_context['post_id'] = $node->nid;
     $page_context['content_uuid'] = isset($node->uuid) ? $node->uuid : '';
-    $account = user_load($node->uid);
-    $page_context['author'] = $account->name;
```
Comment (ynx) [inline acquia_lift.context.inc:84]: No longer surface / collect author information.

File: acquia_lift.module [hunk 2]
```diff
@@ -142,6 +142,9 @@ function acquia_lift_page_build(&$page) {
   $attached['js'][$js_file] = array(
     'type' => 'external',
     'weight' => JS_DEFAULT - 10,
+    // Drupal 7 does not support rendering of 'async' in any non-hacky way,
+    // I am still adding 'async' here in case Drupal 7 supports it in future.
+    'async' => TRUE,
```
Comment (ynx) [inline acquia_lift.module:147]: As commented.

File: acquia_lift.module [hunk 3]
```diff
@@ -153,7 +156,7 @@ function acquia_lift_page_build(&$page) {
 /**
  * Implements hook_node_view().
  */
-function acquia_lift_node_view($node, $view_mode, $langcode) {
+function acquia_lift_node_view($node, $view_mode) {
```
Comment (ynx) [inline acquia_lift.module:159]: Redundant, so cleaning up.

File: tests/acquia_lift.test [hunk 4]
```diff
@@ -136,6 +138,13 @@ class AcquiaLiftsWebTest extends DrupalWebTestCase {
     $this->assertText($submit_confirm_text);
     $this->assertEqual(variable_get('acquia_lift_content_replacement_mode'), 'trusted', 'Content replacement mode was updated to trusted.');
 
+    // Test content origin settings
+    $this->assertEqual(variable_get('acquia_lift_content_origin'), '', 'Content origin data is default to empty.');
+    $edit['acquia_lift_content_origin'] = '08c93130-2e45-45f6-af6d-7c02de8cd90c';
+    $this->drupalPost(NULL, $edit, $submit_button_text);
+    $this->assertText($submit_confirm_text);
+    $this->assertEqual(variable_get('acquia_lift_content_origin'), '08c93130-2e45-45f6-af6d-7c02de8cd90c', 'Content origin was updated.');
```
Comment (ynx) [inline tests/acquia_lift.test:146]: Test form save.

**Review by danikalmar — APPROVED** [review]
looks good...

**Comment by ynx** [comment]
Thank you @danikalmar for the review!

**Pull request merged**